ow_power_system/config/example.cfg

```diff
@@ -0,0 +1,14 @@
+# Prognoser configuration
+model:Battery
+observer:UKF
+predictor: Null
+
+# Observer configuration
+Observer.Q: 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10, 0, 0, 0, 0, 0, 0, 0, 0, 1e-10
+Observer.R: 1e-2, 0, 0, 1e-2
+
+# Predictor configuration
+Model.ProcessNoise: 1e-5, 1e-5, 1e-5, 1e-5, 1e-5, 1e-5, 1e-5, 1e-5
+
+# Load Estimator configuration
+LoadEstimator.Loading: 8
```

Comment:
Is this a file the user should know/care about, or ever edit?  If so, needs documentation.

Reply:
Good point. This configuration file can be used to adjust the calculation technique and model used for prognostics. I will add notes to the README.

Reply:
Added.